### PR TITLE
accountTab added

### PR DESCRIPTION
**Pull request by sergevei**

### CODE REVIEW

File: src/containers/profile/accountTab/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
+@import '../../../scss/shared/breakpoints.scss';
```
Comment (slavasgod) [inline src/containers/profile/accountTab/styles.module.scss]: Remove unused imports

File: src/containers/profile/accountTab/index.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    account,
+    password,
+    buttons_block,
+    payment_btn,
+    payment_btn_block,
+  } = styles;
+
+  return ( 
+    <section className={account}>
+      <section className={password}>
```
Comment (universezzz) [inline src/containers/profile/accountTab/index.jsx]: Please, rewrite wrappers for inputs to the `<div>` . Sections have other semantic purpose.

Comment (universezzz) [reply src/containers/profile/accountTab/index.jsx]: Please wrap all inputs to the `form` tag. This is necessary for a correct semantic structure.

**Review by universezzz — COMMENTED** [review]
.

**Review by universezzz — COMMENTED** [review]
Review completed

**Comment by universezzz** [comment]
> .

File: src/containers/profile/accountTab/index.jsx (hunk 2 above)
Comment (OlegPanasyuk) [reply src/containers/profile/accountTab/index.jsx]: Maybe it necessary to rewrite wrappers by `<article>` tag

**Comment by OlegPanasyuk** [comment]
It seems, that it is better to use routers in the menu of Profile.

**Pull request merged**